.travis.yml

```diff
@@ -7,6 +7,12 @@ python:
   - "3.6"
   - "pypy"
   - "pypy3"
+# Enable 3.7 without globally enabling sudo and dist: xenial for other build jobs
```

Comment:
What's the motivation for this? i.e. why is this required for 3.7, but not wanted for others builds?

Reply:
Nevermind, I see the comment in the PR